File: Sample/src/net/nend/sample/SizeSampleActivity.java
package net.nend.sample;

import android.app.ListActivity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ListView;

public class SizeSampleActivity extends ListActivity {
    
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.size_sample);
    }

    @Override
    protected void onListItemClick(ListView l, View v, int position, long id) {
        super.onListItemClick(l, v, position, id);
        Intent intent = new Intent(getApplicationContext(), SizeActivity.class);
        intent.putExtra("size_type", position);
        startActivity(intent);
    }
}

File: Sample/kotlin/src/main/java/net/nend/sample/kotlin/banner/SizeSampleActivity.kt
package net.nend.sample.kotlin.banner

import android.app.ListActivity
import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.ListView
import net.nend.sample.kotlin.R

class SizeSampleActivity : ListActivity() {

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.size_sample)
    }

    override fun onListItemClick(l: ListView, v: View, position: Int, id: Long) {
        super.onListItemClick(l, v, position, id)

        Intent(applicationContext, SizeActivity::class.java).run {
            putExtra("size_type", position)
            startActivity(this)
        }
    }
}